codec/codec_test.go

```diff
@@ -0,0 +1,25 @@
+package codec
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestIsBetweenNamedFeatureActivationHeight(t *testing.T) {
+	tolerance := int64(5)
+	codecUpgradeKey := ClearUnjailedValSessionKey
+
+	// have to init global map / not mock friendly.
```

Comment:
1. Add a comment at the top that this should never be run in parallel
2. Capture the value of `UpgradeFeatureMap` before the test
3. Run `t.Cleanup(...)` where you reassign the value it had pre-test